tests/common/test_helpers.c

```diff
@@ -78,10 +78,10 @@ void print_data_int(int dev, int32_t data)
         printf("\"data\":[");
         parser_state[dev] |= JSON_STATE_DATA_STARTED;
     }
-    printf("%li", data);
+    printf("%" PRId32, data);
```

Comment:
actual replacement for `li` would be `PRIi32` not `PRId32` don't know if it matters?!

Reply:
First one I saw, OK.

Reply:
force pushed the change.